presto-google-sheets/src/main/java/io/prestosql/plugin/google/sheets/SheetsClient.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.google.sheets;
+
+import com.google.common.collect.ImmutableSet;
+import io.airlift.json.JsonCodec;
+import io.prestosql.spi.PrestoException;
+import io.prestosql.spi.type.Type;
+import io.prestosql.spi.type.VarcharType;
+
+import javax.inject.Inject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import static io.prestosql.plugin.google.sheets.SheetsErrorCode.SHEETS_UNKNOWN_TABLE_ERROR;
+import static java.util.Objects.requireNonNull;
+
+public class SheetsClient
+{
+    private final Set<String> schema;
+    private final SheetsDataProvider sheetsDataProvider;
+
+    @Inject
+    public SheetsClient(SheetsConfig config, JsonCodec<Map<String, List<SheetsTable>>> catalogCodec)
+    {
+        requireNonNull(config, "config is null");
+        requireNonNull(catalogCodec, "catalogCodec is null");
+        this.schema = ImmutableSet.of("default");
+        this.sheetsDataProvider = SheetsDataProvider.getSheetsDataProvider(config);
+    }
+
+    public Set<String> getSchemaNames()
```

Comment:
does it belong here? Maybe it should be moved to `ConnectorMetadata`?

Reply:
ConnectorMetadata delegates to `sheetsClient.getSchemaNames()`
We will use this when we add multiple schema support in this connector,

Reply:
So maybe let's add it then. Future is unknown things may change before you add multiple schema support.

Reply:
Sounds good, I will clear this for now.